packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/fields.tsx

```diff
@@ -199,7 +199,7 @@ const QuickStrategyFields = React.memo(
                                 </div>
                                 {id === 'type-strategy' && (
                                     <div key='description' className='quick-strategy__text'>
-                                        <Text text='sm' weight='normal'>
+                                        <Text size={is_mobile ? 'xxs' : 's'} styles={{ margin: '0 0 2rem' }}>
```

Comment:
can we add a classname here

Reply:
@rupato-deriv no, we have to pass to the Text component 'styles' attribute with new values because in another way defaults class styles will be used. You can see it on the screenshot below.
<img width="525" alt="Screenshot 2022-11-09 at 16 09 10" src="https://user-images.githubusercontent.com/103181650/200838770-8e043ce0-2f7a-497a-ae43-69ec1548b16d.png">